bindata/v3.11.0/openshift-svcat-controller-manager/service-catalog-enabled-rule.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    monitoring-key: service_catalog
+    prometheus: application-monitoring
+    role: alert-rules
+  name: service-catalog-enabled
+  namespace: openshift-service-catalog-controller-manager
+spec:
+  groups:
+    - name: svcat-controller-manager-enabled
+      rules:
+        - alert: ServiceCatalogControllerManagerEnabled
+          annotations:
+            summary: Indicates whether Service Catalog is enabled
+          expr: service_catalog_controller_manager_enabled > 0
+          for: 12h # once a day
```

Comment:
Say I want to see it removed once I remove the service catalog is that possible?

Reply:
I think if I remove the `for:` it will always fire once that condition is met. I didn't know if it would spam without it.

Reply:
Looks like I don't need it. I'll remove it.

Reply:
I think `for:` means "this expression must evaluate to true for X duration before the alert will fire"

https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/#defining-alerting-rules

Reply:
`for:` now I'll leave it off, if it becomes spammy we can edit it.